lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -35,15 +35,23 @@ function collectImageElementInfo() {
     const entries = srcset.split(',');
     const relativeUrls = entries.map(entry => entry.trim().split(' ')[0]);
     return relativeUrls.map(url => {
-      const a = document.createElement('a');
-      a.href = url;
-      return a.href;
+      try {
+        return new URL(url, window.location.href).href;
```

Comment:
> `document.baseURI || window.location.href`

drive-by nit: if there isn't a `<base>` element then these should be equal, and `document.baseURI` will always be defined anyways, so `return new URL(url, document.baseURI).href` should be sufficient